sdk/core/core-http/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 # Release History
 
-## 1.1.9 (Unreleased)
+## 1.1.9 (2020-09-30)
 
+- Add support for XML namespaces in deserialization [PR 11201](https://github.com/Azure/azure-sdk-for-js/pull/11201)
+
+- Add support for NDJSON format [PR 11325](https://github.com/Azure/azure-sdk-for-js/pull/11325)
+
+- Add support for `NO_PROXY` and `ALL_PROXY` environment variables [PR 7069](https://github.com/Azure/azure-sdk-for-js/pull/7069)
```

Comment:
Sorry for crashing late into the party, but what is `ALL_PROXY`?
Does our guidelines expect us to respect this?

Reply:
It's not in the list of https://github.com/Azure/azure-sdk/blob/master/_includes/tables/environment_variables.md, but `ALL_PROXY` can be used if no protocol specific proxy is specified, while protocol specific ones are `HTTP_PROXY` and `HTTPS_PROXY`

Reply:
My concern is that neither does https://github.com/Azure/azure-sdk/blob/1f83516c03e240641de63cc3a9fd176f501183fd/_includes/tables/environment_variables.md talk about ALL_PROXY, nor https://github.com/Azure/azure-sdk/blob/a727537d54a1c71a8edd53d3e582a693b1644d74/docs/general/azurecore.md#proxy-policy

Reply:
Maybe we could submit a PR to add "ALL_PROXY" support as nice to have?

Reply:
Is the concern around whether we should have additional features that are not specified/required by guidelines?

Reply:
Yes. Especially for env variables that the Azure SDKs claim to support, the guidelines are going to be one place of truth to check what we support and we dont. I'd like to see a PR made to the guidelines to make a note of the ALL_PROXY env variable

Reply:
https://github.com/Azure/azure-sdk/pull/1875

I didn't make change to azurecore.md because it already mentioned 
> ... the HTTPS_PROXY (and **associated**) environment variables ...

Reply:
Also it looks like ALL_PROXY support for other languages is under the radar too: https://github.com/Azure/azure-sdk-for-java/issues/5967#issuecomment-659783452